source/patterns/@aws-solutions-constructs/aws-lambda-step-function/lib/index.ts

```diff
@@ -62,23 +62,23 @@ export interface LambdaToStepFunctionProps {
 }
 
 /**
- * @summary The LambdaToStepFunctionProps class.
+ * @summary The LambdaToStepFunctionsProps class.
  */
-export class LambdaToStepFunction extends Construct {
+export class LambdaToStepFunctions extends Construct {
   public readonly lambdaFunction: lambda.Function;
   public readonly stateMachine: sfn.StateMachine;
   public readonly stateMachineLogGroup: logs.ILogGroup;
   public readonly cloudwatchAlarms?: cloudwatch.Alarm[];
 
   /**
-   * @summary Constructs a new instance of the LambdaToStepFunctionProps class.
+   * @summary Constructs a new instance of the LambdaToStepFunctionsProps class.
    * @param {cdk.App} scope - represents the scope for all the resources.
    * @param {string} id - this is a a scope-unique id.
-   * @param {LambdaToStepFunctionProps} props - user provided props for the construct.
+   * @param {LambdaToStepFunctionsProps} props - user provided props for the construct.
    * @since 0.8.0
    * @access public
    */
-  constructor(scope: Construct, id: string, props: LambdaToStepFunctionProps) {
+  constructor(scope: Construct, id: string, props: LambdaToStepFunctionsProps) {
     super(scope, id);
     defaults.CheckProps(props);
```

Comment:
This code in the constructor should all be deleted. A new LambdaToStepfunctionsProps object should be created and populated from the incoming LambdaToStepFunctionProps values. Then the constructor will make a call like:

const wrappedConstruct = new LambdaToStepfunctions(this,  `${id}-wrapped`, newLambdaToStepfunctionsProps);

// After that, the exposed properties of the construct class will need to be populated from the exposed properties of wrapptedConstruct.